ironfist/src/cpp/game/event_loop.cpp

```diff
@@ -0,0 +1,22 @@
+#include "base.h"
+#include "game/game.h"
+#include "registry_prefs.h"
+
+//adding a call to Sleep() during event processing will reduce excessive cpu usage
+//on my machine, 10ms seems to reduce cpu load to <1% while not having any effect
+//on the speed/responsiveness of the game
+
+const int MAX_SLEEP_VALUE = 1000; //sanity check
+
+void __fastcall Process1WindowsMessage_orig();
+void __fastcall Process1WindowsMessage()
+	{

```

Comment:
This one's just a style thing:

The style I've been pushing for in the Ironfist codebase is braces on the same line as the function declaration.  I'm not familiar with any C/C++ style that calls for indenting the brace on the next line. Also, spaces over hard tabs.

I've fought with editor settings every time I upgrade Windows, so I'd be hypocritical to be too hard on this.